dcpy/test/utils/test_versions.py

```diff
@@ -17,12 +19,19 @@ def test_parsing_invalid_version(self):
             versions.parse("23v")
         with self.assertRaises(Exception):
             versions.parse("2v12")
+        with self.assertRaises(Exception):
+            versions.parse("20231212")
 
     def test_bumping_versions(self):
-        for bumped_part, v, v_expected in [
-            ["major", "23v2", "23v3"],
-            ["major", "23v2.1", "23v3"],
-            ["minor", "23v2", "23v2.1"],
-            ["minor", "23v2.1", "23v2.2"],
+        for bumped_part, bump_by, v, v_expected in [
+            ["major", None, "23v2", "23v3"],
+            ["major", None, "23v2.1", "23v3"],
+            ["minor", None, "23v2", "23v2.1"],
+            ["minor", None, "23v2.1", "23v2.2"],
+            [None, 1, "23Q1", "23Q2"],
+            [None, 2, "23Q4", "24Q2"],
+            [None, 7, "23Q2", "25Q1"],
+            [None, 7, "2023-01-01", "2023-08-01"],
```

Comment:
I doubt it'd cause any problems, but this behavior feels a little ambiguous. (would naively expect a 7 day increment, but that's not something we ever do, so...)

Reply:
Yeah that's fair.

For now the only dataset that follows this convention is ztl, which is monthly. Actual dated releases are "20230101". Still not quite ideal - I think we change ztl moving forward to drop the "-01"